tensorboard/backend/path_prefix_test.py

```diff
@@ -62,7 +63,7 @@ def test_bad_path_prefix_with_trailing_slash(self):
 
   def test_empty_path_prefix(self):
     app = path_prefix.PathPrefixMiddleware(self._echo_app, "")
-    server = werkzeug.test.Client(app, werkzeug.BaseResponse)
+    server = werkzeug_test.Client(app, werkzeug.BaseResponse)
```

Comment:
Is `from werkzeug import wrappers; wrappers.BaseReponse` not needed?

Reply:
BaseResponse is aliased correctly. We may want to change this when we upgrade to werkzeug 1.0.

Reply:
Interesting; thanks for investigating.